To solve, run "python run_me.py <image file>" in Terminal

For Instance, python run_me.py maze09.jpg
